src/ui/pages/HardwareWalletConnection/HardwareWalletConnection.tsx

```diff
@@ -92,7 +92,7 @@ export function HardwareWalletConnectionStart({
   const [searchParams] = useSearchParams();
 
   const navigate = useNavigate();
-  const existingAddresses = useAllExistingAddresses();
+  const existingAddresses = useAllExistingMnemonicAddresses();
```

Comment:
What about private addresses here?

Reply:
That's one of the things fixed in the updated logic. Say you imported a single address by private key, but later you're ready to import the whole group. Currently we would show the address as "already added" and block adding it. I think this can be frustrating

Reply:
Ah, wait. I see, this is ledger's import flow. Right, I guess we should use all transacting wallets here